internal/rfc8693/errors.go

```diff
@@ -14,6 +15,9 @@ var (
 	ErrorMissingIss = &ErrorMissingClaim{
 		claim: "iss",
 	}
+
+	// ErrorInvalidClaimCondition represents an error where the claim condition expression is invalid.
+	ErrorInvalidClaimCondition = errors.New("invalid claim condition expression")
```

Comment:
Surprised the go linter didn't complain about this.

```suggestion
	// ErrInvalidClaimCondition represents an error where the claim condition expression is invalid.
	ErrInvalidClaimCondition = errors.New("invalid claim condition expression")
```